lib/tether/tether.go

```diff
@@ -305,6 +317,10 @@ func (t *tether) handleSessionExit(session *SessionConfig) {
 		logs = logs[logCount-MaxDeathRecords+1:]
 	}
 
+	// Remove associated PID file
+	paths := strings.Split(session.Cmd.Path, "/")
+	_ = os.Remove(fmt.Sprintf("%s/%s.pid", PIDFileDir, paths[len(paths)-1]))
```

Comment:
If there is no `/` in the path, paths will be empty and `len(paths)-1` will be `-1`.